client/services/code_preprocessors/PythonCodePreprocessorServiceSpec.js

```diff
@@ -460,23 +462,28 @@ describe('PythonCodePreprocessorService', function() {
   describe('_generateBuggyOutputTestCode', function() {
     it('should add correct buggy output test code to skeleton code',
       function() {
-        var buggyOutputTests = [BuggyOutputTestObjectFactory.create({
+        var buggyOutputTests = [[BuggyOutputTestObjectFactory.create({
```

Comment:
Might want to pull out the [ ] a bit so that it reads a bit cleaner. This is a nit though.